shell/components/Tabbed/index.vue

```diff
@@ -277,8 +276,12 @@ export default {
           :data-testid="`btn-${tab.name}`"
           :aria-controls="'#' + tab.name"
           :aria-selected="tab.active"
+          :aria-label="tab.labelDisplay"
           role="tab"
+          tabindex="0"
           @click.prevent="select(tab.name, $event)"
+          @keyup.enter="select(tab.name, $event)"
+          @keyup.space="select(tab.name, $event)"
```

Comment:
NOTE: Anchor tags that are treated as buttons are the symptom of requiring the pattern of enforcing `space` to be defined. Let's continue to handle these the way that we are, but I'm creating #13174 so that we can follow-up in the future to address. I think that all of the accessibility work will be a good starting point to identifying these instances.